account_product_fiscal_classification/model/account_product_fiscal_classification.py

```diff
@@ -93,7 +126,7 @@ def _compute_product_tmpl_info(self):
     def write(self, vals):
         res = super(AccountProductFiscalClassification, self).write(vals)
         pt_obj = self.env['product.template']
-        if 'supplier_tax_ids' in vals or 'customer_tax_ids' in vals:
+        if 'purchase_tax_ids' in vals or 'sale_tax_ids' in vals:
```

Comment:
It has been fixed in an other PR #33 . I guess you should undo this changes and rebase your PR for more visibility.